### PR TITLE
Update gnome-shell-extension-caffeine.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: caffeine@patapon.info/locale/it_IT/LC_MESSAGES/gnome-shell-extension-caffeine.po [hunk 1]
```diff
@@ -98,43 +95,36 @@ msgid "Never"
...
-#, fuzzy
 msgid "Enable notifications when Caffeine is enabled or disabled"
-msgstr "Mostra notifiche quando abilitato/disabilitato"
+msgstr "Abilita le notifiche quando Caffeine è abilitata o disabilitata"
 
 #: preferences/displayPage.js:98
-#, fuzzy
 msgid "Status indicator Position"
-msgstr "Mostra indicatore nel pannello in alto"
+msgstr "POsizione indicatore di stato"
```
Comment (eonpatapon) [inline caffeine@patapon.info/locale/it_IT/LC_MESSAGES/gnome-shell-extension-caffeine.po:123]:
```suggestion
msgstr "Posizione indicatore di stato"
```

Comment (albanobattistella) [reply caffeine@patapon.info/locale/it_IT/LC_MESSAGES/gnome-shell-extension-caffeine.po]: Hi,
done!
A.

**Comment by stuarthayhurst** [comment]
It might be worth updating the locales on `master`, 2 of the strings changed with #220

**Comment by eonpatapon** [comment]
@albanobattistella can you rebase? there's been a few changes in the locales

**Comment by pakaoraki** [comment]
Replaced by #259

**Pull request closed without merging**